packages/mux-uploader/src/mux-uploader-sr-text.ts

```diff
@@ -0,0 +1,59 @@
+import { globalThis, document } from 'shared-polyfills';
+import { getMuxUploaderEl } from './utils/element-utils';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+
+.sr-only {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
+</style>
+
+<div class="sr-only" id="sr-only" aria-live="polite"></div>
+`;
+
+class MuxUploaderSrTextElement extends globalThis.HTMLElement {
```

Comment:
question(non-blocking): Do we need the SR-only element to be its own component? For all of the visual components, I can think of plausible use cases, but this feels more tightly coupled to the uploader element itself?

Reply:
this kind of ties into the nested shadow doms issue where if we wanted a .sr-only utility class we don't have an easy way of sharing it. Both since we can't include it at the top level of an element and expect it to be available to nested shadow doms, but also, we don't currently have a way to do CSS includes (though, I believe this is solvable via esbuild). Though, that could mean we're including the same utility multiple times.

Reply:
good callout @cjpillsbury. I could be convinced either way, but decided to move it to its own bc I plan to handle more events and didn't want to clutter up the `mux-uploader` file with that logic

If either of you feel strongly that it's better off tied to the uploader el, lmk and I can move it back in and treat this PR as a spike on eventing.

Reply:
definitely not a blocker for this PR. We can revisit if/when appropriate between now and v1.0